src/neuroconv/tools/testing/data_interface_mixins.py

```diff
@@ -120,6 +131,33 @@ def test_conversion_as_lone_interface(self):
                 self.run_custom_checks()
 
 
+def _create_mock_probe(*, num_channels: int):
```

Comment:
Would it be possible to extend this function to handle `num_groups`? So it constructs a fake multi-shank probe for testing in conjunction with the other interfaces?

cc @samuelgarcia @alejoe91 @h-mayorquin

Reply:
I exposed the num_shanks parameter, and it defaults to 3.